crates/test-collector/src/lib.rs

```diff
@@ -94,6 +98,12 @@ pub enum ExpectedTestResult {
     Panics(ExpectedPanicValue),
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub enum Fork {
```

Comment:
`ForkConfig`  is imo a better name. Fork suggest it storing fork data

Reply:
but on the other hand:
```
pub enum ForkConfig {
    Id(String),
    Config(String, BlockId),
}
```
it will create ForkConfig::Config imo it's not looks good

Reply:
I could change `Config` -> `Params`
```
pub enum ForkConfig {
    Id(String),
    Params(String, BlockId),
}
```